notebooks/DemoSimpleInterface.ipynb

```diff
@@ -37,12 +37,7 @@
    },
    "outputs": [],
    "source": [
-    "!pip install pytorch-lightning==1.3.3\n",
-    "# !pip install cloud-tpu-client==0.10 https://storage.googleapis.com/tpu-pytorch/wheels/torch_xla-1.8.1-cp37-cp37m-linux_x86_64.whl\n",
-    "!pip install scGeneFit\n",
-    "!pip install scanpy\n",
-    "!pip install anndata\n",
-    "!pip install lassonet"
+    "!pip install markermap"
```

Comment:
Installing our package with pip will automatically install these as dependencies? Really cool!

Reply:
Yeah, it will install all the packages in the `install_requires` field of setup.cfg: https://github.com/Computational-Morphogenomics-Group/MarkerMap/blob/6c7efcaea60180a04f5cac80ef5925d93e8f0a58/setup.cfg

In the future, we could split the package so that you can download a version that doesn't include all the other models (LassoNet, RankCorr, and Smashpy), so there would be a lot less dependencies if you just wanted to run MarkerMap.